dashboard/config/initializers/script_preload.rb

```diff
@@ -0,0 +1,2 @@
+# Preload scripts before application fork
+Script.script_cache
```

Comment:
OK, disabled the script cache preloader when running a rake task not invoked via Spring. This allows e.g. `rake db:setup` to work, while still allowing `bundle exec spring rake -T` to use the preloaded app.